upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -482,6 +476,7 @@ func (c *ApplyClusterCmd) Run() error {
 		}
 
 		role := ig.Spec.Role
+		glog.V(4).Infof("Generating renderNodeUpConfig for %s", role)
```

Comment:
I did not intend to checkin this. It was added for debugging. Will remove this log,